pino.js

```diff
@@ -82,10 +82,13 @@ function pino (...args) {
   const time = (timestamp instanceof Function)
     ? timestamp : (timestamp ? epochTime : nullTime)
 
+  const extreme = stream.minLength === 4096
```

Comment:
I think this make sense if stream is a SonicBoom instance instead.

Reply:
then there's no need for noop - which is a little less explicit  - what if both? 

```
const extreme = stream instanceof SonicBoom && stream.minLength === 4096
```

Reply:
right, we need to check both